INSTALL.md

```diff
@@ -45,8 +45,7 @@ $ wget -O /tmp/go${VERSION}.${OS}-${ARCH}.tar.gz https://dl.google.com/go/go${VE
 Finally, set up your environment for Go:
 
 ```
-$ echo 'export GOPATH=${HOME}/go' >> ~/.bashrc && \
-  echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc && \
+$ echo 'export PATH=/usr/local/go/bin:${PATH}' >> ~/.bashrc && \
```

Comment:
if we don't set GOPATH, it's default value is $HOME/go:

```
$ unset GOPATH
$ go env GOPATH
/home/mem/go
```

this means `go install` will install binaries there, and that means $PATH should still contain that.

Maybe change the above to the following?

```
$ echo 'export PATH=/usr/local/go/bin:$(go env GOPATH)/bin:${PATH}' >> ~/.bashrc && \
```

Reply:
Seconding this - it's nice to help out people who aren't familiar with go by putting this in.